pkg/karmadactl/promote.go

```diff
@@ -100,11 +103,8 @@ type CommandPromoteOption struct {
 	// Namespace is the namespace of legacy resource
 	Namespace string
 
-	// ClusterContext is the cluster's context that we are going to join with.
-	ClusterContext string
-
-	// ClusterKubeConfig is the cluster's kubeconfig path.
-	ClusterKubeConfig string
```

Comment:
Why delete `ClusterContext` and `ClusterKubeConfig`?

Reply:
we can get `ClusterContext `and `ClusterKubeConfig` from clusterinfo directly

Reply:
We keep it for the time being and it could be removed in the future.